include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -669,21 +760,25 @@ private :
 
         if (Bisection)
         {
-            bisection<CT>(lon1, lat1, lon2, lat2, lon3, lat3,
-                          spheroid, res12.distance/2, res12.azimuth, result);
+            bisection(lon1, lat1, lon2, lat2, lon3, lat3, spheroid, 
+                res12.distance/2, res12.azimuth, result);
```

Comment:
Ok so I see that in this file the formatting is consistent and it's the last version. I prefer the one that was here used before the change but this is ok too. The question is which one should we use in general?

Reply:
So the question is align below `(` or change line and use tab, right?

Reply:
We do both in practice. In the past I often used a tab (= spaces) but many people and IDE's now align at the `(`